Doc/library/stdtypes.rst

```diff
@@ -1759,7 +1759,14 @@ expression support in the :mod:`re` module).
 .. method:: str.capitalize()
 
    Return a copy of the string with its first character capitalized and the
-   rest lowercased.
+   rest lowercased. For example::
+
+      >>> 'PYTHON IS AMAZING'.capitalize()
+      'Python is amazing'
+      >>> 'Ǉjubljana starts With a non-English digraph'.capitalize()
+      'ǈjubljana starts with a non-english digraph'
```

Comment:
Probably just a mis-click, let's try again (and with "non-ASCII"):

```suggestion
      >>> 'ǋemačka starts With a non-ASCII digraph'.capitalize()
      'ǋemačka starts with a non-ascii digraph'
```

Reply:
It worked. Thank you.

Reply:
I've tried every digraph listed here: https://en.wikipedia.org/wiki/Digraph_(orthography)#In_Unicode

And all of them have failed.

![WhatsApp Image 2025-10-01 at 17 08 25](https://github.com/user-attachments/assets/0cf9c0f1-bfe8-4220-b75a-a020742d98fb)